recipes/triqs/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: triqs
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/triqs/releases/download/{{ version }}/triqs-{{ version }}.tar.gz
+  sha256: 5f7fca3dd140d86e464ab18ff91fa16febad865344ec2c5b924434f507ef4038
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - boost
+    - fftw
+    - gmp
+    - hdf5
+    - h5py
+    - mako
+    - matplotlib
```

Comment:
Let's move the discussion here a bit. So, have you tried dropping this? Let's see if this fixes the conflict? Do you actually need the pyqt frontend? That's one of the things matplotlib provides over matplotlib-base.

Reply:
If nothing else works, just do a binary search over the dependencies here to find the source of the conflict. It's silly of course, but it should only take a few iterations.

Reply:
We require the following imports in our matplotlib wrapper: https://github.com/TRIQS/triqs/blob/2.2.x/pytriqs/plot/mpl_interface.py#L25-L28

If this imports are contained in `matplotlib-base` then we should just give it a shot.

Reply:
```python
import matplotlib as mpl
import matplotlib.pyplot as plt
from matplotlib import rc
```

Reply:
@saraedum Locally I removed matplotlib from my environment, leaving only matplotlib-base. The above imports fail in that case (the first one, actually).

Reply:
I meant whether this would fix the conflict at least. You need to understand where this conflict comes from exactly first. Then we can make things work. :)

Reply:
(it does not matter if you pile up a tons of trivial commits here btw. It's normal to try out all kinds of things when packaging something until you make the CI happy.)

Reply:
giving it a go